run_md.py

```diff
@@ -0,0 +1,879 @@
+        else:
+            for wdir_md in wdir_to_continue_list:
+                for f in glob(os.path.join(wdir_md, '#*#')):
+                    os.remove(f)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='''Run or continue MD simulation.\n
+    Allowed systems: Protein in water, Protein-Ligand-multiple Coenzymes, Protein-Ligand, Protein-multiple Coenzymes''')
+    parser.add_argument('-p', '--protein', metavar='FILENAME', required=False, type=filepath_type,
+                        help='input file of protein. Supported formats: *.pdb or gro')
+    parser.add_argument('-d', '--wdir', metavar='WDIR', default=None, type=filepath_type,
+                        help='Working directory. If not set the current directory will be used.')
+    parser.add_argument('-l', '--ligand', metavar='FILENAME', required=False, type=filepath_type,
+                        help='input file with compound. Supported formats: *.mol or sdf or gro')
+    parser.add_argument('--cofactor', metavar='FILENAME', default=None, type=filepath_type,
+                        help='input file with compound. Supported formats: *.mol or sdf or gro')
+    parser.add_argument('--gromacs_version', metavar='STRING', default='GROMACS/2021.4-foss-2020b-PLUMED-2.7.3',
+                        help='gromacs_version')
+    parser.add_argument('--not_add_H', action='store_true', default=False,
+                        help='disable to add hydrogens to molecules before simulation.')
+    parser.add_argument('--clean_previous_md', action='store_true', default=False,
+                        help='remove all previous prepared for the current system MD files.\n'
+                             'Prepared ligand and protein files will be intact and reused if it exists.')
+    parser.add_argument('--hostfile', metavar='FILENAME', required=False, type=str, default=None,
+                        help='text file with addresses of nodes of dask SSH cluster. The most typical, it can be '
+                             'passed as $PBS_NODEFILE variable from inside a PBS script. The first line in this file '
+                             'will be the address of the scheduler running on the standard port 8786. If omitted, '
+                             'calculations will run on a single machine as usual.')
+    parser.add_argument('-c', '--ncpu', metavar='INTEGER', required=False, default=cpu_count(), type=int,
+                        help='number of CPU per server. Use all cpus by default.')
+    parser.add_argument('--topol', metavar='topol.top', required=False, default=None, type=filepath_type,
+                        help='Required if gro file of the protein is provided')
+    parser.add_argument('--topol_itp', metavar='topol_chainA.itp topol_chainB.itp', required=False, nargs='+', default=None, type=filepath_type,
+                        help='if protein has multiple chain there are multiple topologies for each of them. Itp files for each chain are required')
```

Comment:
better
itp files for individual protein chains.

Are file names important to be properly recognized or not? If so, this should be explicitly stated in the help message

Reply:
all files automatically written in topology by gmx. So here should be exact names and files gmx pdb2gmx returns

Reply:
this should be reflected in the help message